it.mijorus.gearlever.json

```diff
@@ -0,0 +1,73 @@
+{
+    "app-id" : "it.mijorus.gearlever",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gearlever",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+
+        "--filesystem=xdg-data/applications:rw",
+        "--filesystem=xdg-data/icons:ro",
+        "--filesystem=home:rw",
```

Comment:
This is of course moot exeercise with the access to host commands, but why does it need full write access instead of using filesystem portal?

Reply:
This is the only way I know about to write files inside the user's folder.

The user can specify a custom path for storing appimages, so this is why I'm specifying full access to the home directory